curriculum/challenges/english/04-data-visualization/data-visualization-with-d3/change-styles-based-on-data.md

```diff
@@ -22,10 +22,9 @@ The `style()` method is not limited to setting the `color` - it can be used with
 
```

Comment:
Line 17 comment in example code. Is this in the dictionary?

Reply:
Nope - I just missed it 😅